docs/ios-manual-testing.md

```diff
@@ -0,0 +1,69 @@
+---
+id: ios-manual-testing
+title: Scope iOS Agent Manual testing
+sidebar_label: Manual testing
+---
+
+Scope allows to do manual test runs that automatically connect to your Scope instance. You can run your application from Xcode in your simulator or device, but you can also distribute a build to your testers with the functionality ready.
+
+The results will appear in your local development scratchpad in Scope
+
+## Installation
+
+Link your application target with `ScopeAgent`
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Cocoapods-->
+By adding the pod to your `Podfile` and running `pod install`. For example:
+
+```
+target 'MyApp' do
+ pod 'ScopeAgent'
+end
+```
+
+or
+
+```
+target 'MyFramework' do
+ pod 'ScopeAgent'
+end
+```
+
+<!--Carthage-->
+Add the `ScopeAgent` dependency to your `Cartfile` if not already done in previous steps, and run `cart update`
+
+```
+binary "https://releases.undefinedlabs.com/scope/agents/ios/ScopeAgent.json"
+```
+
+In your application or framework targets, add `ScopeAgent.framework` located in `Carthage/Build/iOS` to the
+*Linked frameworks and Libraries* in General target settings or to the *Link Binaries With Libraries* build phase. 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+Then you must add the following entries in your application Info.plist
```

Comment:
This step depends on whether the user wants to hardcode the values for distribution or not. I would put first the instructions of the env vars for local Xcode runs, and later, as an optional step, update the `Info.plist` if the user wants to distribute the application with manual testing activated.

Reply:
Changed